README.md

```diff
@@ -15,47 +15,35 @@ Add the plugin and repositories listing to `project.clj`. **NB: You need to add
 :plugins [[s3-wagon-private "1.2.0"]]
 ```
 
-You can store credentials either in an encrypted file or as
-environment variables. For the encrypted file, add this to
-`project.clj`:
+To authenticate to the S3 bucket, you can either:
 
-```clj
-:repositories [["private" {:url "s3p://mybucket/releases/" :creds :gpg}]]
-```
+- Use any of the AWS SDK [chained provider class][chained-provider-class] credential providers. An excerpt of the most commonly used credential providers:
+     - Environment Variables - `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
+     - Java System Properties - aws.accessKeyId and aws.secretKey
```

Comment:
Does master currently support `~/.aws/credentials` ? I'm not having any luck with it.